packages/flutter/test/widgets/widget_inspector_test.dart

```diff
@@ -419,6 +420,7 @@ class _TestWidgetInspectorService extends TestWidgetInspectorService {
           child: WidgetInspector(
             key: inspectorKey,
             exitWidgetSelectionButtonBuilder: exitWidgetSelectionButtonBuilder,
+            moveExitWidgetSelectionButtonBuilder: null,
```

Comment:
can we add a test case for moving the button left and right using the arrow button?

Reply:
Yes, added!